### PR TITLE
Remove routing decorators in `test_routing.py`

### PR DESCRIPTION
**Pull request by aminalaee**
Subset of #1481 drops usage of routing decorators in tests/test_routing.py .

### CODE REVIEW

File: tests/test_routing.py [hunk 1]
```diff
@@ -54,86 +54,91 @@ async def async_ws_endpoint(cls, websocket: WebSocket):
...
+            endpoint=path_with_parentheses,
+            name="path-with-parentheses",
+        ),
+        WebSocketRoute("/ws", endpoint=websocket_endpoint),
+        # WebSocketRoute("/ws/{room}", endpoint=websocket_params),
+    ]
+)
+
+
 @app.websocket_route("/ws/{room}")
```
Comment (aminalaee) [inline tests/test_routing.py]: Need this for coverage in `websocket_route`.

Comment (lovelydinosaur) [reply tests/test_routing.py]: Gotcha. No surprises there.

A sensible approach would be to *first* issue a pull request adding the "We no longer document this decorator style API, and its usage is discouraged" docstrings noted in https://github.com/encode/starlette/issues/1481 and also adding `pragma: nocover` to those decorator methods.

Comment (lovelydinosaur) [reply tests/test_routing.py]: Okay - you ought to be able to add the last one in as well, now.

**Review by lovelydinosaur — CHANGES_REQUESTED** [review]
Once `websocket_params` is also in, this looks good.

File: tests/test_routing.py (hunk 1 above)
Comment (aminalaee) [reply tests/test_routing.py]: Thanks, done.

**Review by lovelydinosaur — APPROVED** [review]

**Comment by lovelydinosaur** [comment]
Fab 🌟

**Pull request merged**